docs/es/docs/contributing.md

```diff
@@ -0,0 +1,447 @@
+    $ source ./env/bin/activate
+    ```
+
+    </div>
+
+=== "Windows PowerShell"
+
+    <div class="termy">
+
+    ```console
+    $ .\env\Scripts\Activate.ps1
+    ```
+
+    </div>
+
+=== "Windows Bash"
+
+    O si usas Bash para Windows (por ejemplo, <a href="https://gitforwindows.org/" class="external-link" target="_blank">Git Bash</a>):
+
+    <div class="termy">
+
+    ```console
+    $ source ./env/Scripts/activate
+    ```
+
+    </div>
+
+Para verificar que funcionó, usa:
+
+=== "Linux, macOS, Windows Bash"
+
+    <div class="termy">
+
+    ```console
+    $ which pip
+
+    some/directory/fastapi/env/bin/pip
+    ```
+
+    </div>
+
+=== "Windows PowerShell"
+
+    <div class="termy">
+
+    ```console
+    $ Get-Command pip
+
+    some/directory/fastapi/env/bin/pip
+    ```
+
+    </div>
+
+Si muestra el binario `pip` en `env/bin/pip`, entonces funcionó. 🎉
+
+Asegúrate de tener la última versión de pip en su entorno local para evitar errores en los siguientes pasos:
+
+<div class="termy">
+
+```console
+$ python -m pip install --upgrade pip
+
+---> 100%
+```
+
+</div>
+
+!!! tip "Consejo"
+    Cada vez que instales un nuevo paquete con `pip` en ese entorno, activa el entorno nuevamente.
+
+    Esto asegura que si usas un programa de la terminal instalado por ese paquete, usa el de tu entorno local y no cualquier otro que pueda instalarse globalmente.
+
+### Instalar requerimientos utilizando pip
+
+Después de activar el entorno como se describe arriba:
+
+<div class="termy">
+
+```console
+$ pip install -r requirements.txt
+
+---> 100%
+```
+
+</div>
+
+Esto instalará todas las dependencias y tu FastAPI local en tu entorno local.
+
+### Utilizando tu FastAPI local
+
+Si creas un archivo Python que importa y usa FastAPI, y lo ejecutas con Python desde tu entorno local, usará tu código fuente local clonado de FastAPI.
+
+Y si actualizas ese código fuente local de FastAPI cuando vuelvas a ejecutar ese archivo Python, utilizará la versión nueva de FastAPI que acabas de editar.
+
+De esa manera, no tendrás que "instalar" tu versión local para poder probar cada cambio.
+
+!!! note "Detalles Técnicos"
+    Esto solo sucede cuando realizas la instalación utilizando el `requirements.txt` incluido en lugar de ejecutar `pip install fastapi` directamente.
+
+    Esto se debe a que dentro del archivo `requirements.txt`, la versión local de FastAPI está marcada para instalarse en modo "editable", con la opción `-e`.
+
+### Formatear el código
+
+Hay un script que puedes ejecutar y que formateará y limpiará todo tu código:
+
+<div class="termy">
+
+```console
+$ bash scripts/format.sh
+```
+
+</div>
+
+También ordenará automáticamente todas tus importaciones.
+
+Para ordenarlos correctamente, necesitas tener FastAPI instalado localmente en tu entorno, con el comando de la sección anterior usando `-e`.
+
+## Documentación
+
+Primero, asegúrese de configurar su entorno como se describe anteriormente, eso instalará todos los requisitos.
```

Comment:
Previously you use `requerimientos` for `requirements` ([here](https://github.com/tiangolo/fastapi/pull/10965/files#diff-95a846f155a6570f1c255067c29ccdbd0a784effd0d46d318de93dad472301c6R104)). To be consistent, we need to use the same translation here.

```suggestion
Primero, asegúrese de configurar su entorno como se describe anteriormente, eso instalará todos los requerimientos.
```